services/schedview/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
Unless you are using the k8s API from within the application, consider getting ride of this.